### DISCUSSION TITLE
extend the scope of the SIM Swap API

### DISCUSSION THREAD

**Discussion by Ma-Guang-Han** [camaraproject/SimSwap#158] [Ideas]
Dear community, this is Chunghwa Telecom. Our company believes that for the SIM Swap API, adding the last network usage time of the original SIM card as an output would enhance the accuracy of fraud detection. If a SIM card is still using the network one second and is swapped for another SIM card the next second, it might indicate fraudulent activity.

**Comment by bigludo7** [comment]
Hello @Ma-Guang-Han 
Thanks for sharing the proposal
Could you elaborate a bit more about how you envision this feature? for example what will be the attribute is the request/response?
For you it should be a separate API from the current one ?

Depending on your answers perhaps we will have to go thru API backlog.

Adding a couple of usual suspects for this APi @AxelNennker @fernandopradocabrillo @gregory1g

**Comment by Ma-Guang-Han** [comment]
Hi @bigludo7 

I believe an additional output can be added to the SIM Swap retrieve-date API. The input would remain as phoneNumber, and in addition to the existing output of latestSimChange (the time the status was last checked), a new output could be added: latestNetworkUsage (the last network usage time of the original SIM card). The parameter type would follow RFC 3339 and must include a timezone string ($date-time). I believe having these two timestamps will allow for a more accurate determination of potential fraud activity.

**Reply by gregory1g** [reply]
Hello, @Ma-Guang-Han 

>If a SIM card is still using the network one second and is swapped for another SIM card the next second, it might indicate fraudulent activity.

According to our knowledge the strongest fraud indicator is  the time between SimSwap and some action towards the application service provider. Like if the SimSwap happened few minutes before one tries to transfer a lot of money using OTP SMS confirmation (or tries to recover the forgotten password using phone related channel) this looks like a fraud.

Meanwhile if one lost their phone or it was stolen or one migrates from sim to eSIM or from one eSIM to another, etc -  the old SIM can still be active when SimSwap request is done. Therefore, it looks like that for an AS, the recent network activity of the previous SIM does not really add a valuable information. 

Can you explain how an ASP can use "last network activity of the previous SIM" information to improve fraud identification (when it knows time of user's activity and time of sim swap event already)?

Side note: "latestNetworkUsage" for an API taking phoneNumber as an identifier, references recent network usage of this number, not a "previous SIM card which was associated with this phone number". And for a multi-sim case "previous" is not a correct word either.
